database/news_schema.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const newsSchema = mongoose.Schema({
+  newsTitle: {type: String, required: true},
```

Comment:
wouldn't it make our code more readable in the hbs files ? cause event schema also has title and date ....??